src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -8,6 +8,9 @@
 #include "MSStoreInstallerHandler.h"
 #include "WorkflowBase.h"
 
+#include <winget/NameNormalization.h>
+#include <winget/Manifest.h>

```

Comment:
These should be in pch.h?

Reply:
Without actual measurement, I have come to the conclusion that we should only put external code in the pch by default.  Internal code ends up causing a lot of build churn when something changes.  So for something that is used in almost every file, that should go in the pch.  But for something like NameNormalization, it should stay in the specific cpp files that use it.